spring-cloud-skipper/spring-cloud-skipper/src/main/java/org/springframework/cloud/skipper/domain/CloudFoundryApplicationManifestReader.java

```diff
@@ -53,10 +53,10 @@ public class CloudFoundryApplicationManifestReader implements SkipperManifestRea
 	public List<CloudFoundryApplicationSkipperManifest> read(String manifest) {
 		if (canSupport(manifest)) {
 			List<CloudFoundryApplicationSkipperManifest> applicationSpecs = new ArrayList<>();
-			YAMLMapper mapper = new YAMLMapper();
-			mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
-			mapper.configure(MapperFeature.ACCEPT_CASE_INSENSITIVE_PROPERTIES, true);
-			mapper.setPropertyNamingStrategy(PropertyNamingStrategy.KEBAB_CASE);
+			YAMLMapper mapper = YAMLMapper.builder().
+				configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false).
+				configure(MapperFeature.ACCEPT_CASE_INSENSITIVE_PROPERTIES, true).build();
+			mapper.setPropertyNamingStrategy(PropertyNamingStrategies.KEBAB_CASE);
```

Comment:
There is a method on builder for naming strategy as well.

Reply:
Nice Catch!